common/src/main/kotlin/io/homeassistant/companion/android/common/data/servers/ServerManagerImpl.kt

```diff
@@ -51,6 +52,7 @@ class ServerManagerImpl @Inject constructor(
     private val authenticationRepos = mutableMapOf<Int, AuthenticationRepository>()
     private val integrationRepos = mutableMapOf<Int, IntegrationRepository>()
     private val webSocketRepos = mutableMapOf<Int, WebSocketRepository>()
+    private val connectionRepos = mutableMapOf<Int, ServerConnectionStateProvider>()
```

Comment:
So... why did you not call it a ConnectionRepository?

Reply:
Fair, it is only a manager/helper (we don't have a good name for that), but the (internal) variable is still calling it a repository 😉